src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -325,9 +325,144 @@ module.exports = class LevelPlane extends Array {
       posToRefresh.push(this.redstoneListON[i]);
     }
 
+    //once we're done updating redstoneWire states, check to see if doors should open/close
```

Comment:
This method is getting pretty complex — can we start separating out concerns into different functions?